__tests__/app.test.js

```diff
@@ -308,6 +308,32 @@ describe("/api/articles/:article_id/comments", () => {
   });
 });
 
+describe("GET /api/users", () => {
+  test("GET: 200 - responds with an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users.length).toBeGreaterThan(0);
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET: 404 - When given a misspelt URL /api/usrs responds with msg: 'invalid URL", () => {
```

Comment:
Don't really need this as it's caught by your generic bad URL test (you should have one somewhere else?)